Week01/Problem04/klai_04.py

```diff
@@ -0,0 +1,13 @@
+def isPalindrome(n):
+  l = [int(i) for i in str(n)]

```

Comment:
This line is redundant.  If `type(n)` is `int`, then `str(n)` already is a list, since strings are treated by Python as lists of characters.  So, your `isPalindrome` function could be re-written more compactly as
```
def isPalindrome(num):
    if str(num) == str(num)[::-1]:
        return True
    else:
        return False
```
where I've used list slicing to reverse the order of the string by `str(num)[::-1]`.